app.py

```diff
@@ -0,0 +1,120 @@
+from flask import Flask, request, jsonify, render_template
+import json
+from jsonschema import validate
+import jsonschema
+import psycopg2
+from psycopg2.extras import RealDictCursor
+
+app = Flask(__name__)
+# run_with_ngrok(app)
+
+POSTGRESQL_URI = 'postgres://axnbpeci:rADWtHAG9X0UDVyrJyvenm8-uX4GjZmR@tyke.db.elephantsql.com/axnbpeci'
+# POSTGRESQL_URI='postgres://postgres:mysecretpassword@postgres/postgres'
+connection = psycopg2.connect(POSTGRESQL_URI)
+
+try:
+    with connection:
+        with connection.cursor() as cursor:
+            cursor.execute("CREATE TABLE users (name VARCHAR(43), surname VARCHAR(43), email VARCHAR(43) UNIQUE NOT NULL, id SERIAL PRIMARY KEY);")
+except psycopg2.errors.DuplicateTable:
+    pass
+
+#Loading schema to a variable
+def load_schema():
+    with open('schema.json') as f:
+        schema = json.load(f)
+    return schema
+
+def add_user_db(data):
+    try:
+        with connection:
+            with connection.cursor() as cursor:
+                sql = 'INSERT INTO users (name, surname, email) VALUES ( %s, %s, %s)'
+                val = (data['name'], data['surname'], data['email'])
+                cursor.execute(sql, val)
+        return data
+    except psycopg2.errors.UniqueViolation:
+        return "Error: Email already exists"
+
+def load_all_users_db():
+    with connection:
+        with connection.cursor() as cursor:
+            cursor = connection.cursor(cursor_factory=RealDictCursor)
+            cursor.execute("SELECT * FROM users ORDER BY id;")
+            users = cursor.fetchall()
+        return users
+
+def find_user_db(id):
+    with connection:
+        with connection.cursor() as cursor:
+            cursor = connection.cursor(cursor_factory=RealDictCursor)
+            cursor.execute(f"SELECT * FROM users WHERE id = {id};")
+            user = cursor.fetchall()
+        return user
+
+def delete_user_db(id):
+    with connection:
+        with connection.cursor() as cursor:
+            cursor = connection.cursor(cursor_factory=RealDictCursor)
+            cursor.execute(f"DELETE FROM users WHERE id = {id};")
+        return 'Success'
+
+def update_user_db(id, data):
```

Comment:
Next up would be to separate all these DB functions into a separate file. The same goes for the endpoint handlers. This way the code will be clearer and easier to read.